Resources/Prototypes/_DV/Entities/Objects/Fun/toys.yml

```diff
@@ -85,3 +85,45 @@
   - type: Sprite
     sprite: _DV/Objects/Fun/Toys/mortyplush.rsi
     state: icon
+
+- type: entity
+  parent: BasePlushie
+  id: PlushieIPC
+  name: ipc plushie
+  description: A massive robo-plushie. The screen might be hard plastic, but the rest of it is soft and cuddly!
+  components:
+  - type: Sprite
+    sprite: _DV/Objects/Fun/Toys/ipcplush.rsi
+    layers:
+    - state: icon
+    - state: heart
+      shader: unshaded
+    - state: screenshine
+  - type: MultiHandedItem
+  - type: Fixtures
+    fixtures:
+      fix1:
+        density: 20
```

Comment:
this removes the fixture shape you have to copy paste the original from BaseItem or whatever